### PR TITLE
Fixup build on qt > 5.14

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/qusbmoded.cpp [hunk 1]
```diff
@@ -399,7 +407,11 @@ void QUsbModed::updateSupportedModes(QString aModes)
 
 void QUsbModed::updateAvailableModes(const QString &aModes)
 {
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
     const QStringList result = aModes.split(',', QString::SkipEmptyParts);
+#else
+    const QStringList result = aModes.split(',', Qt::SkipEmptyParts);
+#endif
```
Comment (pvuorela) [inline src/qusbmoded.cpp:414]: I'd rather check versions with qt 5.14 which introduced the new api. Doesn't need qt 6.

**Review by pvuorela — COMMENTED** [review]
There now being a bunch of -qt6 library versions on PRs, I'm not entirely sure should we go that direction everywhere and reintroduce two separate versions when we've just gotten rid of the qt4 leftovers. Also pondering how you are using this since the .spec is still qt5-only. Building with some alternative system?

Suppose the -qt6 vs -qt5 matters more when the different versions need to be parallel installable. At least on sailfish this is only needed by few system components so not sure yet if that parallel installability would be ever needed. Not a suggestion yet, just thinking aloud, but one alternative could be just renaming the whole thing as -qt. Or then uglier option could be to start by compiling the -qt5 library with qt 6.

For the qt 5.14 #ifdeffing I would have no objections. Could also just start by having that first.

**Comment by neochapay** [comment]
> There now being a bunch of -qt6 library versions on PRs, I'm not entirely sure should we go that direction everywhere and reintroduce two separate versions when we've just gotten rid of the qt4 leftovers. Also pondering how you are using this since the .spec is still qt5-only. Building with some alternative system?

Yeap for nemomobile

File: src/src.pro [hunk 2]
```diff
@@ -32,15 +32,15 @@ com_meego_usb_moded.files = $$USB_MODED_INCLUDE_PATH/com.meego.usb_moded.xml
 com_meego_usb_moded.header_flags = -N -c QUsbModedInterface
 com_meego_usb_moded.source_flags = -N -c QUsbModedInterface
 
-headers.path = $$INSTALL_ROOT$$PREFIX/include/usb-moded-qt5
+headers.path = $$INSTALL_ROOT$$PREFIX/include/usb-moded-qt$${QT_MAJOR_VERSION}
```
Comment (pvuorela) [inline src/src.pro]: These changes don't now match the title of the PR. Could just skip these now.

**Review by pvuorela — APPROVED** [review]

**Pull request merged**